scripts/VisionSuite/blobLocator.py

```diff
@@ -11,10 +11,56 @@
 if this file is run.
 
 """
+import os, cv2
+import numpy as np
+from utils import ColorFilter
 
+def blobLocator(img,
+    visualize=False,
+    setThresholds=False,
+    bounds = {}):
 
-def blobLocator(img):
-	return (10,20),30
+    #Assume failure
+    error = True
+
+    # Manually adjust thresholding
+    if setThresholds:
+        pass
+
+    # Filter image
+    grey = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
+
+    # Apply SimpleBlobDetector
+    sbd = cv2.SimpleBlobDetector()
+    blobs = sbd.detect(grey)
+
+    # TODO: Visualize
```

Comment:
I'm hoping to build in a visualizer in addition to the accuracyChecker's visualizer